### PR TITLE
internal/rpc: fix broken link

### PR DESCRIPTION
**Pull request by roylee17**
remove a DOT

### CODE REVIEW

File: src/content/intro/getting-started/first-abci-app.vue [hunk 1]
```diff
@@ -95,16 +95,12 @@ asked for a hash, or committed the state. The result of <code>commit</code> is j
...
 To avoid wasting resources on invalid transactions,
 ABCI provides the <code>check_tx</code> message,
-which application developers can use to accept or reject messages,
+which application developers can use to accept or reject transactions,
 before they are stored in memory or gossipped to other peers.</p>
 <p>In this instance of the counter app, <code>check_tx</code> only allows transactions whose integer is greater than the last committed one.</p>
 <p>Let&#x2019;s kill the console and the dummy application, and start the counter app:</p>
 <pre><code>counter
 </code></pre>
-<p>Again, the code is just</p>
```
Comment (melekes) [inline src/content/intro/getting-started/first-abci-app.vue]: why is this removed?

File: src/content/docs/guides/app-dev.vue [hunk 2]
```diff
@@ -44,7 +44,7 @@
 <p>To use ABCI in your programming language of choice, there must be a ABCI server in that language.
 Tendermint supports two kinds of implementation of the server:</p>
 <ul>
-<li>Asynchronous, raw socket server</li>
+<li>Asynchronous, raw socket server (Tendermint Socket Protocol, also known as TSP or Teaspoon)</li>
```
Comment (melekes) [inline src/content/docs/guides/app-dev.vue]: TSP is applicable to both socket and grpc servers, no?

**Comment by melekes** [comment]
Thank you for working on this 👍

**Comment by nylira** [comment]
Hi, please make content changes only in in the Markdown files within the `./content/` directory. Any content changes you make in the Vue files will be overwritten by `npm run docs`.

**Comment by roylee17** [comment]
Obsoleted changes.
